sdk/schemaregistry/azure-schemaregistry-avroencoder/tests/test_avro_encoder.py

```diff
@@ -433,13 +433,14 @@ def test_parse_record_name(self, **kwargs):
         with pytest.raises(InvalidSchemaError):
             sr_avro_encoder.encode({"name": u"Ben"}, schema=schema_invalid_name_in_fullname) 
 
-        schema_invalid_name_reserved_type = """{
-            "name":"record",
-            "type":"record",
-            "fields":[{"name":"name","type":"string"}]
-        }"""
-        with pytest.raises(InvalidSchemaError):
-            sr_avro_encoder.encode({"name": u"Ben"}, schema=schema_invalid_name_reserved_type)
+        # Commenting out as this passes with avro==1.11.0, but fails with avro==1.11.1
+        #schema_invalid_name_reserved_type = """{
```

Comment:
Is this a known issue? May be worthwhile to open a github issue... or is it an expected behaviour?

Reply:
opened an issue here: https://github.com/Azure/azure-sdk-for-python/issues/26180
thank you!

Reply:
Looks like this was [an intentional decision](https://issues.apache.org/jira/browse/AVRO-3370) to only reserve primitive type names in avro (aligning with Java's avro behavior). Changing this test, so that it raises only for primitive type name. Bumping avro min dependency.